core/CronArchive.php

```diff
@@ -1001,6 +1000,11 @@ private function logArchiveTimeoutInfo()
             $this->logger->info("- Archiving was last executed without error "
                 . $this->formatter->getPrettyTimeFromSeconds($dateLast, true) . " ago");
         }
+
+
+        if ($this->websiteIdArchiveList->isContinuingPreviousRun()) {
```

Comment:
@diosmosis not too important since we don't have it for any of the other methods in those classes but might make sense to add an interface to the sharedsiteids and fixedsiteids just to make sure those things are implemented (or could check if the method exists)